terraform/modules/prometheus_all/resources.tf

```diff
@@ -51,4 +51,5 @@ module grafana {
   admin_password           = var.grafana_admin_password
   json_dashboards          = var.grafana_json_dashboards
   extra_datasources        = var.grafana_extra_datasources
+  influxdb_credentials     = module.influxdb.credentials
```

Comment:
On a plan, it's failing with 
```
Error: Unsupported attribute

  on .terraform/modules/prometheus_all/terraform/modules/prometheus_all/resources.tf line 54, in module "grafana":
  54:   influxdb_credentials     = module.influxdb.credentials

This value does not have any attributes.
```

Probably fixable with
```
  influxdb_credentials     = module.influxdb[0].credentials
```

Reply:
changed